cmd/node.go

```diff
@@ -1,3 +1,20 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
```

Comment:
If you rename file there is no need to update copyright years. Copy right years indicates when the file and it's content was created initially. If somebody pastes this code into it's own library he will not update years too.

Reply:
Thanks. Here my IDE accidentally added double header. In other noted places Github not some reason wrongly indicates moved files as new ones.